test/test_jit.py

```diff
@@ -1106,8 +1120,10 @@ def forward(self, x):
                       if x.startswith('_observer_')])
         assert len(dtypes) == 2, 'Expected to have 2 different types of dtype'
 
-    @_tmp_donotuse_dont_inline_everything
-    def test_insert_quant_dequant(self):
+    @given(
+        is_per_channel=st.booleans()
+    )
```

Comment:
Can we replace it with `for is_per_channel in [True, False]` and not pull hypothesis here?

Reply:
isn't this cleaner?